lnwire/short_channel_id.go

```diff
@@ -37,3 +42,11 @@ func (c *ShortChannelID) ToUint64() uint64 {
 	return ((uint64(c.BlockHeight) << 40) | (uint64(c.TxIndex) << 16) |
 		(uint64(c.TxPosition)))
 }
+
+// String generates a string representation of the channel ID. This returns the
```

Comment:
Returning this encoding doesn't really convey any meaning when displayed. I've thought a bit about how best to display this (instead of just the integer everywhere), how about something along the lines of:
```
height:index:pos
```
or
```
height/index/pos
```